### PR TITLE
Add action to ensure dist is up to date

### PR DESCRIPTION
**Pull request by dyladan**
image

### CODE REVIEW

File: src/utils.ts
Comment (dyladan) [inline src/utils.ts:203]: Required to fix the build with latest typescript

**Comment by trask** [comment]
can we wait to merge this until after the cherry-picks are merged? (#15)

File: package.json
Comment (dyladan) [inline package.json:31]: Some of the dependency updates aren't specifically needed but I think they're all a good idea

**Comment by dyladan** [comment]
> can we wait to merge this until after the cherry-picks are merged? (#15)

sure

File: .github/workflows/build-dist.yml
Comment (trask) [inline .github/workflows/build-dist.yml:3]: (e.g. so it won't apply on dependabot branches)

```suggestion
  push:
    branches:
      - main
```

**Review by trask — APPROVED** [review]

File: src/main.ts
Comment (dyladan) [reply src/main.ts:26]: yes

**Pull request merged**
